From f7e3a9ddf71f28c4ef4bb9f137d68d313951f0f0 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Thu, 10 Nov 2022 15:49:22 +0100
Subject: [PATCH 0523/1695] dt-bindings: add PWR dt-bindings

Add PWR dt bindings.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 8ee4d30f85c7c62dcd6a0623b5fdffc5566c0243 dt-bindings: add PWR dt-bindings]
Change-Id: I6d98e12ae9577372a944e044773f46670de9144b
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/358366
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 .../dt-bindings/power/stm32mp1-power.h        | 20 +++++++++++++++++++
 1 file changed, 20 insertions(+)
 create mode 100644 core/include/dt-bindings/power/stm32mp1-power.h

diff --git a/core/include/dt-bindings/power/stm32mp1-power.h b/core/include/dt-bindings/power/stm32mp1-power.h
new file mode 100644
index 000000000..fe394a826
--- /dev/null
+++ b/core/include/dt-bindings/power/stm32mp1-power.h
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: GPL-2.0 or BSD-3-Clause */
+/*
+ * Copyright (C) STMicroelectronics 2018 - All Rights Reserved
+ * Author: Yann Gautier <yann.gautier@st.com> for STMicroelectronics.
+ */
+
+#ifndef DT_BINDINGS_STM32MP1_POWER_H
+#define DT_BINDINGS_STM32MP1_POWER_H
+
+#define STM32_PM_CSLEEP_RUN			0
+#define STM32_PM_CSTOP_ALLOW_STOP		1
+#define STM32_PM_CSTOP_ALLOW_LP_STOP		2
+#define STM32_PM_CSTOP_ALLOW_LPLV_STOP		3
+#define STM32_PM_CSTOP_ALLOW_LPLV_STOP2		4
+#define STM32_PM_CSTOP_ALLOW_STANDBY_DDR_SR	5
+#define STM32_PM_CSTOP_ALLOW_STANDBY_DDR_OFF	6
+#define STM32_PM_SHUTDOWN			7
+#define STM32_PM_MAX_SOC_MODE			8
+
+#endif /* DT_BINDINGS_STM32MP1_POWER_H */
-- 
2.47.2

